inc/model.class.php

```diff
@@ -520,6 +527,14 @@ public function rawSearchOptions()
             'datatype'      => 'bool',
         ];
 
+        $tab[] = [
+            'id'            => 87,
+            'table'         => $this->getTable(),
+            'field'         => 'multiline_value_replace',
+            'name'          => __('Multiline field value replace', 'datainjection'),
```

Comment:
```suggestion
            'name'          => __('Replacing the value of multiline text fields', 'datainjection'),
```
should be the same label as the option in plugin configuration